src/dvr/dvr_inotify.c

```diff
@@ -63,7 +63,11 @@ pthread_t dvr_inotify_tid;
 
 void dvr_inotify_init ( void )
 {
+#if ENABLE_ANDROID
+  _inot_fd = inotify_init();
```

Comment:
Please, define inotify_init1 in tvheadend.h - there's already android block - like:
<pre>
#define inotify_init1(a) inotify_init()
</pre>